app/models/manageiq/providers/ibm_terraform/inventory/parser/configuration_manager.rb

```diff
@@ -14,17 +14,37 @@ def configuration_profiles
     end
   end
 
+  # Sample IaasResources - virtual_machines response from  API
+  # [{
+  #     "type": "virtual_machine",
+  #     "name": "aws_instance.web-server",
+  #     "stackId": "5ee8dd8b6143d40017c2ee3c",
+  #     "provider": "Amazon EC2",
+  #     "idFromProvider": "i-0d59174316cafb2x0",
+  #     "typeFromProvider": "aws_instance",
+  #     "module_path": [
+  #         "root"
+  #     ],
+  #     "consolelinks": [
+  #         "https://console.aws.amazon.com/ec2/v2/home"
+  #     ],
+  #     "ipaddresses": [
+  #         "18.215.xxx.yyy",
+  #         "172.31.xx.yy"
+  #     ],
+  #     "tainted": false,
+  #     "tenantId": "fbb9ab6a-b54d-43d4-9200-6d950700588f",
+  #     "namespaceId": "acme",
+  #     "namespaceMapping": true,
+  #     "id": "5ee8ddde6143d40017c2ee45"
+  # }]
   def configured_systems
-    collector.stacks.each do |stack|
-      template_ref = stack["templateId"].to_s
-
+    collector.virtual_machines.each do |virtual_machine|
       persister.configured_systems.build(
-        :manager_ref           => stack["id"].to_s,
-        :hostname              => stack["name"],
-        :last_checkin          => stack["created_at"],
-        :build_state           => stack["build"],
-        :mac_address           => stack["status"],
-        :configuration_profile => persister.configuration_profiles.lazy_find(template_ref)
+        :manager_ref          => virtual_machine["id"].to_s,
+        :name                 => virtual_machine["name"] + ' - ' + virtual_machine["ipaddresses"].to_s,
```

Comment:
Why tack on the IP addresses to the name?

Reply:
Name we have today is not unique to identify the vm(something like - aws_instance.web-server). We can look into returning hostname or some other more unique name from CAM side.

Reply:
Okay, name doesn't have to be unique so unless it is common to have lots of IaaS resources with the same name we don't need to try to make each name unique here.  Ideally we should show the name which they'd see in CAM so there is consistency.

Reply:
Addressed this in latest PR

Reply:
:+1: